frontend/interfaces/config.ts

```diff
@@ -118,6 +118,7 @@ export interface IConfig {
     org_logo_url: string;
   };
   sandbox_enabled: boolean;
+  sandbox_expiry?: string;
```

Comment:
I wouldn't consider this to be part of the `config` interface since it comes from a separate API. I think it might be cleaner for it to live in a new `sandbox` interface?

Reply:
yep i forgot to remove it. i took it out.